crypto/blake2/blake2.c

```diff
@@ -59,6 +59,19 @@ static uint64_t blake2b_load(const uint8_t block[BLAKE2B_CBLOCK], size_t i) {
   return CRYPTO_load_u64_le(block + 8 * i);
 }
 
+static void copy_digest_words_to_dest(uint8_t* dest, uint64_t* src, size_t word_count) {
+#ifdef OPENSSL_BIG_ENDIAN
+  for(size_t i = 0; i < word_count; i++) {
+    uint64_t word = src[i];
+    for(size_t j = 0; j < sizeof(uint64_t); j++) {
+      dest[i * sizeof(uint64_t) + j] = (word >> (j * 8)) & 0xFF;
+    }
+  }
```

Comment:
Can this use CRYPTO_store_u64_le?
```suggestion
  for(size_t i = 0; i < word_count; i++) {
    CRYPTO_store_u64_le(dest[i * sizeof(uint64_t)], src[i]
  }
```
I thought about combining the big and little endian versions, but right now the single memcpy is more efficient and simpler to understand.

Reply:
Updated in next commit.